lg69t/README.md

```diff
@@ -27,7 +35,8 @@ python3 firmware_tool.py --port=/dev/ttyUSB1 --gnss=/path/to/lg69t_teseo_A.B.CC.
 
 Replace `/dev/ttyUSB1` with the serial port connected to the device via UART1 (use the appropriate COM port number in Windows, e.g., `Standard COM Port`).
 
-Replace `/path/to/quectel-lg69t-am-0.XX.0_upg.bin` and `/path/to/lg69t_teseo_A.B.CC.D_sta.bin` with the path to the application image and GNSS image files respectively. The device must be rebooted after each command is executed successfully.
+If upgrading via a `p1fw` file, replace `/path/to/quectel-lg69t-am-0.XX.0.p1fw` with the path to the `p1fw` file. If upgrading via
```

Comment:
I would have the section on how to specify the path together with your explanation of the parameter. Like:

```
To update via `p1fw` path:

python3 firmware_tool.py --port=/dev/ttyUSB1 --p1fw=/path/to/quectel-lg69t-am-0.XX.0.p1fw

Replace `/path/to/quectel-lg69t-am-0.XX.0.p1fw` with the path to the `p1fw` file.
```

Instead of separating that explanation on specifying the file path.